python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java

```diff
@@ -193,9 +193,10 @@ public void package_inner_submodules_symbols() {
   @Test
   public void package_relative_import() {
     Map<String, Symbol> osSymbols = symbolsForModule("os");
-    // TODO: Add imported symbols SONARPY-938
-    // Symbol sysSymbol = osSymbols.get("sys");
-    // assertThat(sysSymbol.kind()).isEqualTo(Kind.AMBIGUOUS);
+    Symbol sysSymbol = osSymbols.get("sys");
+    assertThat(sysSymbol.kind()).isEqualTo(Kind.OTHER);
+    Set<String> sysExportedSymbols = symbolsForModule("sys").keySet();
+    assertThat(((SymbolImpl) sysSymbol).getChildrenSymbolByName().values()).extracting(Symbol::name).containsAll(sysExportedSymbols);
```

Comment:
Do you think we should eventually move to actually use a `ModuleSymbol` type for this?
Certainly not in this PR, but for longer term I'm not sure.

Reply:
Indeed, I think it would be cleaner to have `ModuleSymbol` in the long run! Although I already tried once and I found some unexpected problems: https://jira.sonarsource.com/browse/SONARPY-658